llvm/include/llvm/CodeGen/TargetRegisterInfo.h

```diff
@@ -489,6 +489,12 @@ class TargetRegisterInfo : public MCRegisterInfo {
   virtual const MCPhysReg*
   getCalleeSavedRegs(const MachineFunction *MF) const = 0;
 
+  /// Return a null-terminated list of all of the callee-saved registers on
+  /// this target when IPRA is on. Normally, this list should be null.
```

Comment:
Please expand on this comment. The list should include any non-allocatable registers that the backend uses and assumes will be saved by all calling conventions. This is typically the ISA-standard frame pointer, but could include the thread pointer, TOC pointer, or base pointer for different targets.

Reply:
Thanks review! addressed in https://github.com/llvm/llvm-project/pull/109597/commits/ef0f2682c9d8866a2a1fb552cb5941dbf416be56.